src/resty/aws/request/execute.lua

```diff
@@ -22,6 +22,7 @@ local function execute_request(signed_request)
     port = signed_request.port,
     scheme = signed_request.tls and "https" or "http",
     ssl_server_name = signed_request.host,
+    ssl_verify = false, -- TODO: added after updating lua-resty-http which now defaults to "true" and breaks stuff, needs to be fixed!!
```

Comment:
This needs to be fixed; figure out how to set the proper CA store, or why it doesn't work if the CA store is correct

Reply:
In the Kong CLI the CA bundle hasn't been set yet. It generates the bundle and then starts the process while using it, but the CLI itself doesn't have it set.

Reply:
fix could be to restart the CLI with the CA bundle set, so looping the CLI to restart itself, similar to this: https://github.com/Kong/lua-resty-aws/blob/main/spec/resty-runner.lua